### DISCUSSION TITLE
NetBeans

### DISCUSSION THREAD

**Discussion by lssyhnk** [apache/netbeans#7706] [Q&A]
Ich suche jemanden der sich gut mit Netbeans auskennt. Ich komme mit meiner Aufgabe nicht weiter. Sämtliche Fehlermeldungen etc. Bitte um Hilfe :D

**Comment by ramiswailem** [comment]
Schreib einfach deine Fragen...

**Reply by Chris2011** [reply]
Hey @lssyhnk, this is a public repo, please do this in english. Thx.
